packages/ui/badge.tsx

```diff
@@ -10,11 +10,11 @@ interface Props {
 
 export function Badge({children, variant = 'blue', testId}: Props) {
   const variantClassName = classNames(
-    'flex flex-row items-center rounded py-1 px-1',
+    'rounded p-1 flex flex-row items-center',
     {
       'bg-gray-medium text-xs text-gray-900': variant === 'transparent',
       'text-sm bg-stone-100': variant === 'gray',
-      'bg-blueGrey-100 text-blueGrey-900 text-sm': variant === 'blue',
+      'bg-consortiumBlue-100 text-neutral-900 text-sm': variant === 'blue',
```

Comment:
Old issue: is it possible to align our custom names with the Tailwind convention? E.g. by using `bg-consortium-blue` (with dashes)?